packages/@aws-cdk/aws-logs/lib/cross-account-destination.ts

```diff
@@ -43,7 +61,26 @@ export interface CrossAccountDestinationProps {
  *
  * @resource AWS::Logs::Destination
  */
-export class CrossAccountDestination extends cdk.Resource implements ILogSubscriptionDestination {
+export class CrossAccountDestination extends CrossAccountDestinationBase {
+  /**
+   * Import an existing CloudWatch Logs Destination given its ARN.
+   *
+   * @param scope The parent creating construct (usually `this`).
+   * @param id The construct's name.
+   * @param destinationArn AWS CloudWatch Logs Destination ARN (i.e. arn:aws:logs:<region>:<account-id>:destination:MyDestination).
+   */
+  public static fromDestinationArn(scope: Construct, id: string, destinationArn: string): ILogSubscriptionDestination {
+    const stack = cdk.Stack.of(scope);
+    const destinationName = stack.splitArn(destinationArn, ArnFormat.COLON_RESOURCE_NAME).resourceName!;
+
+    class Import extends CrossAccountDestinationBase {
```

Comment:
I don't think we need the base class at all, `Import` can just extend `Resource` and implement `ILogSubscriptionDestination` like the `Import` in `fromRoleArn`: https://github.com/comcalvi/aws-cdk/blob/c97810359a3c00758a8e23621be53e9a7d204a7d/packages/%40aws-cdk/aws-iam/lib/role.ts#L218.

Reply:
Yeah, I was torn between the two ways of representing imported resources.

The one benefit of using a `Base` class (and why I went that route) is that the `bind()` functions are equivalent between the imported and created resources. I'll change this back.